src/commands/clone.js

```diff
@@ -128,13 +128,13 @@ export async function clone(opts, destinationDir) {
   let appDir = path.join(destinationDir, directoryName);
 
   if (opts.force) {
-    await spinify(rmrf(appDir), `Destroying directory ${directoryName}`);
+    await spinify(rmrf(appDir), `Destroying directory ${directoryName}.`);
```

Comment:
I'm wondering - if we're in "spinify", should we add `...` instead of `.`?

Reply:
Makes sense to me, will look for other instances of spinify messages with periods.

Reply:
Please just check whether that looks okay. I'm not sure for how long these messages are shown.